.github/workflows/main.yml

```diff
@@ -777,7 +793,8 @@ jobs:
     - name: Install Python dependencies
       run: python3 -m pip install ruamel.yaml
     - name: Install library dependencies with Conda (Windows)
-      run: mamba install -q yaml-cpp mkl highfive
+      # fmt needs to match the version of the windows-2022 runner
+      run: mamba install -q yaml-cpp mkl highfive fmt=8.1
```

Comment:
It would be helpful to mention that specifically, the version needs to match the windows-2022 runner that is selected to upload `cantera_shared.dll`.